vendor/holidays/lib/generated_definitions/REGIONS.rb

```diff
@@ -1,6 +1,6 @@
 # encoding: utf-8
 module Holidays
-  REGIONS = [:ar, :at, :au, :au_act, :au_nsw, :au_nt, :au_qld, :au_qld_brisbane, :au_qld_cairns, :au_sa, :au_tas, :au_tas_north, :au_tas_south, :au_vic, :au_vic_melbourne, :au_wa, :be_fr, :be_nl, :bg_bg, :bg_en, :br, :br_sp, :br_spcapital, :ca, :ca_ab, :ca_bc, :ca_mb, :ca_nb, :ca_nl, :ca_ns, :ca_nt, :ca_nu, :ca_on, :ca_pe, :ca_qc, :ca_sk, :ca_yt, :ch, :ch_ag, :ch_ai, :ch_ar, :ch_be, :ch_bl, :ch_bs, :ch_fr, :ch_ge, :ch_gl, :ch_gr, :ch_ju, :ch_lu, :ch_ne, :ch_nw, :ch_ow, :ch_sg, :ch_sh, :ch_so, :ch_sz, :ch_tg, :ch_ti, :ch_ur, :ch_vd, :ch_vs, :ch_zg, :ch_zh, :cl, :co, :cr, :cz, :de, :de_bb, :de_be, :de_bw, :de_by, :de_by_augsburg, :de_by_cath, :de_hb, :de_he, :de_hh, :de_mv, :de_ni, :de_nw, :de_rp, :de_sh, :de_sl, :de_sn, :de_sn_sorbian, :de_st, :de_th, :de_th_cath, :dk, :ecbtarget, :ee, :el, :es, :es_an, :es_ar, :es_ce, :es_cl, :es_cm, :es_cn, :es_ct, :es_ex, :es_ga, :es_ib, :es_lo, :es_m, :es_mu, :es_na, :es_o, :es_pv, :es_v, :es_vc, :federalreserve, :federalreservebanks, :fedex, :fi, :fr, :fr_a, :fr_m, :gb, :gb_con, :gb_eaw, :gb_eng, :gb_gsy, :gb_iom, :gb_jsy, :gb_nir, :gb_sct, :gb_wls, :ge, :gg, :hk, :hr, :hu, :id, :ie, :im, :in, :is, :it, :it_bl, :it_fi, :it_ge, :it_pd, :it_rm, :it_ro, :it_to, :it_tv, :it_ve, :it_vi, :it_vr, :je, :jp, :kr, :kz, :li, :lt, :lu, :lv, :ma, :mt_en, :mt_mt, :mx, :mx_pue, :my, :nerc, :ng, :nl, :no, :nyse, :nz, :nz_ak, :nz_ca, :nz_ch, :nz_hb, :nz_mb, :nz_ne, :nz_nl, :nz_ot, :nz_sc, :nz_sl, :nz_ta, :nz_we, :nz_wl, :pe, :ph, :pl, :pt, :pt_li, :pt_po, :ro, :rs_cyrl, :rs_la, :ru, :sa, :se, :sg, :si, :sk, :th, :tn, :tr, :ua, :unitednations, :ups, :us, :us_ak, :us_al, :us_ar, :us_az, :us_ca, :us_co, :us_ct, :us_dc, :us_de, :us_fl, :us_ga, :us_gu, :us_hi, :us_ia, :us_id, :us_il, :us_in, :us_ks, :us_ky, :us_la, :us_ma, :us_md, :us_me, :us_mi, :us_mn, :us_mo, :us_ms, :us_mt, :us_nc, :us_nd, :us_ne, :us_nh, :us_nj, :us_nm, :us_nv, :us_ny, :us_oh, :us_ok, :us_or, :us_pa, :us_pr, :us_ri, :us_sc, :us_sd, :us_tn, :us_tx, :us_ut, :us_va, :us_vi, :us_vt, :us_wa, :us_wi, :us_wv, :us_wy, :ve, :vi, :za]
```

Comment:
Not sure why the lists are so different 🤔 

Can you do a quick diff between the old and new to see what has been added/removed?

Reply:
Sure

Reply:
These regions are missing `us_az, us_co, us_oh, us_or, us_sd, us_wy`. 

It appears they were only ever explicitly used for one one holiday. It was updated here https://github.com/discourse/discourse-calendar/commit/3e67f009c1fca577521e2c00a52756744f43861e

As to why it's now showing up as a change here, perhaps the definitions weren't generated for that change?

Reply:
Nice catch. Yeah, we probably forgot to generate/update the definitions.

Looks like those regions are still referenced in `assets/javascripts/lib/regions.js`, which needs to be updated using the `rake javascript:update_constants` task.

Can you do that as well?

Reply:
Sure

Reply:
Done